api/github.js

```
@@ -142,116 +157,108 @@ export default async function handler(req) {
+
     try {
       const q = encodeURIComponent(`repo:${repo} label:"good first issue" state:open`);
       const res = await fetchWithFallback(
         `https://api.github.com/search/issues?q=${q}&per_page=30&sort=created&order=desc`,
         { headers: ghHeaders }
       );
-      if (!res.ok) {
-        return new Response(JSON.stringify({ total: 0, items: [], error: `GitHub ${res.status}` }), { status: 200, headers });
-      }
-      const data = await res.json();
+      const responseHeaders = addRateLimitHeaders(res, { ...headers });
```

ESLint

[error] 'addRateLimitHeaders' is not defined.

(no-undef)